### ISSUE TITLE
Name parameter still checks ID on Get-Series

### ISSUE THREAD

**Issue by Yevrag35** [Yevrag35/PoshSonarr#18]
### OBSERVED
```powershell
Get-SonarrSeries 1        # returns series with the ID '1' .
Get-SonarrSeries -Name 1  # returns series with the title '1'.
Get-SonarrSeries -N 1     # returns series with the ID '1'.
```
To allow ID's to be passed in from parameter position 1, the <code>-Name</code> parameter validates if the incoming value can be converted to <code>[int]</code>.  If it can, the value will be treated as a series ID, instead of a series title.

There is an exception to this rule.  If the Cmdlet detects that it was called with the explicit <code>-Name</code> parameter declared, the value is treated as the series' title even if it can converted to <code>[int]</code>.

However, PowerShell allows for automatic parameter aliases to be created.  This means that unless the full name of the parameter, <code>-Name</code>, is specified, the exception is not followed.

### EXPECTED
The Cmdlet should detect any of the possible aliases for <code>-Name</code> in addition to the full parameter name to additionally trigger the exemption.
